crates/storage/storage-api/src/state.rs

```diff
@@ -42,54 +41,6 @@ pub trait StateProvider:
     + Send
     + Sync
 {
-    /// Get storage of given account.
```

Comment:
now that we've removed all functions from this type, we can add a blanket impl for T that satisfies all traits, then we dont need to impl StateProvider manually anymore